gen3/bin/kube-setup-system-services.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Run terraform as a script for updates on EKS.
+# This should not be ran as a crontjob, it might cause services disruption, so be carefull
+#
+# Cases in which this script might be useful:
+#  1.) EKS reseleased a new minor version and said release came along with a new AMI that terraform picks up.
+#        This case should not be disruptive at all
+#  2.) A change on the user-data script for the worker nodes, (this MUST be tested before bulk update)
+#  3.) Addition/Deletion of ops_team keys [cloud-automation/files/authorized_keys/ops_team]
+#
+
```

Comment:
Create three new constant variables with some default value, for example:
KUBE_PROXY_VERSION="1.15.11"
COREDNS_VERSION="1.6.6"
CNI_VERSION="1.6"

and replace their values if passed as arguments during the execution of the script, like if you do something like `gen3 kube-setup-system-services kube-proxy=1.20 coredns=1.8 cni=1.9`.

That would be useful in the future.